paddlets/ensemble/ensemble_forecaster_base.py

```diff
@@ -32,9 +32,9 @@ class EnsembleForecasterBase(BaseModel, metaclass=abc.ABCMeta):
     def __init__(self,
                  in_chunk_len: int,
                  out_chunk_len: int,
-                 skip_chunk_len: int = 0,
-                 estimators: List[Tuple[object, dict]] = None,
-                 verbose=False
+                 skip_chunk_len: Optional[int] = 0,
```

Comment:
Line35 skip_chunk_len should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional